plugins/modules/lightsail_snapshot.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: lightsail_snapshot
+version_added: "1.0.0"
+short_description: Creates snapshots of AWS Lightsail instances
+description:
+    - Creates snapshots of AWS Lightsail instances
+author:
+    - "Nuno Saavedra (@Nfsaavedra)"
+options:
+  state:
+    description:
+      - Indicate desired state of the target.
+    default: present
+    choices: ['present', 'absent']
+    type: str
+  region:
+    description: AWS region in which to create the instance snapshot.
+    required: true
+    type: str
+  snapshot_name:
+    description: Name of the new instance snapshot
```

Comment:
```suggestion
    description: Name of the new instance snapshot.
```

Minor niggle - https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-fields

> description: Must be written in full sentences, in other words, with capital letters and periods/full stops.

vs 

> short_description: Unlike description:, short_description should not have a trailing period/full stop